index.js

```diff
@@ -0,0 +1,155 @@
+/* eslint-disable */
+const BASE_URL = 'https://trektravel.herokuapp.com/trips';
+const continents = [];
+const getTrips = (url) => {
+  $.get(url, (response) => {
+    console.log('success!');
+    console.log(response);
+    response.forEach((trip) => {
+      const tripInfo = `<li data-id= ${trip.id} id = "trip-${trip.id}"><strong>${trip.name}</strong> - ${trip.weeks} week(s) in ${trip.continent} </li>`;
+      $('#trip-list ul').append(tripInfo);
+      if (!continents.includes(trip.continent.toLowerCase()) && trip.continent) {
+        continents.push((trip.continent).toLowerCase());
+      };
+    });
+  }).fail(() => {
+    $('#message').html('<h3> No trips found :( )</h3>');
+  }).always(() => {
+      console.log('Adenture awaits you!');
+  });
+};
+const addTrip = () => {
+  // TODO: Add "Cancel" button to clear add new trip form
+  const newTripForm = `
+  <form id= "new-trip-form" action="https://trektravel.herokuapp.com/trips" method="post">
+  <h3> ADD NEW TRIP </h3>
+  <section>
+    <div class = "form-input">
+      <label>Name:</label>
+      <input type="text" id="name" name="name"></input>
+    </div>
+    <div class = "form-input">
+      <label>Continent:</label>
+      <input type="text" id="continent" name="continent"></input>
+    </div>
+    <div class = "form-input">
+      <label>About:</label>
+      <input type="textfield" id="about" name="about"></input>
+    </div>
+    <div class = "form-input">
+      <label>Category:</label>
+      <input type="text" id="category" name="category"></input>
+    </div>
+    <div class = "form-input">
+      <label>Weeks:</label>
+      <input type="integer" id="weeks" name="weeks"></input>
+    </div>
+    <div class = "form-input">
+      <label>Cost:</label>
+      <input type="float" id="cost" name="cost"></input>
+      </section>
+    </div>
+  <section class="button">
+    <button type="submitReservation">Add This Trip</button>
+  </section>
+  </form>`;
+  $('#trip-list').prepend(newTripForm);
```

Comment:
This particular form doesn't have any HTML that needs to be dynamically generated. Since the form will look the same in all cases, you may consider having this on the html already, and just using jQuery to hide()/show() it when appropriate